packages/react-utilities/src/compose/getSlots.test.tsx

```diff
@@ -4,6 +4,7 @@ import { nullRender } from './nullRender';
 import { ComponentState } from './types';
 
 describe('getSlots', () => {
+  // eslint-disable-next-line react/require-default-props -- dummy component for testing
```

Comment:
after bumping versions of eslint-plugin-react and airbnb, lint rules started to properly work.

did some troubleshooting:
- airbnb didn't change definition of this rule
- plugin-react doesn't specify any changes to default rule-set based on their [changelog](https://github.com/yannickcr/eslint-plugin-react/blob/master/CHANGELOG.md)

Reply:
update -> https://github.com/microsoft/fluentui/pull/18808#discussion_r665453895